openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,62 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' not in data:
+            return
+        if 'load' in data['resources']:
+            extra_values = {
+                'load_1': data['resources']['load'][0],
+                'load_5': data['resources']['load'][1],
+                'load_15': data['resources']['load'][2],
+            }
+            metric, created = Metric._get_or_create(
+                object_id=pk,
+                content_type=ct,
+                key='load',
+                field_name='load',
+                name='Resources: Load',
+            )
+            metric.write(
+                data['resources']['load'][1] / data['resources']['cpus'],
+                extra_values=extra_values,
+            )
+            if created:
+                self._create_resources_graph(metric, resource='load')
+                self._create_resources_threshold(metric, resource='load')
+        if 'disk' in data['resources']:
+            used_bytes, size_bytes = 0, 0
+            for disk in data['resources']['disk']:
+                used_bytes += disk['used_bytes']
+                size_bytes += disk['size_bytes']
```

Comment:
```
used_percent = 0
for disk in data['resources']['disk']:
    used_percent += disk['used_percent']
used_percent = used_percent / len(data['resources']['disk']) / 100
```

Reply:
@PabloCastellano, the `df` command currently outputs this on my **VM**
```
Filesystem           1K-blocks      Used Available Use% Mounted on
/dev/root               258016     18792    233984   7% /
tmpfs                   121960        76    121884   0% /tmp
/dev/sda1                16112      3872     11916  25% /boot
tmpfs                      512         0       512   0% /dev
```
Now if you observe the used percent represents percentage usage of that individual filesystem and does not represent the `used_percent` of the whole system, am I missing something here?

Reply:
You are right, that approach is better. Nevermind!